### ISSUE TITLE
Update documentation page with new sections and descriptions of new features

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#283]
From Reviewer 1 (#278):

" It is not so clear that the force algorithm is disabled on specific nodes once that node has been manually relocated to a new position. This may be described under the help section, but I could not find it although it is mentioned in the manuscript (line 217)."

It is mentioned in the Documentation page, under the section how GRNsight displays the graph, third to last bullet point.

It is good feedback that it wasn't seen in the Documentation page, so I'm thinking that making more smaller sections with subheadings will make the page more readable.

**Comment by kdahlquist** [comment]
OK, @dondi, here are the collapsible sections that I want on the Documentation page (sub sections are denoted by 1a, 2b, etc.)

Default to opening section 1 (but not the subsections to 1) with all the other sections closed.  This will essentially give an outline for the help.

1. Formatting the input file

    [leave first two bullet points under this main heading]

    1a. Creating a GRNsight input Excel workbook

    [the rest of the current text should go here up to the YEASTRACT part]

    1b. Formatting a YEASTRACT regulation matrix for use with GRNsight

    [contains current text about YEASTRACT]

    1c. Formatting a Cytoscape SIF file for use with GRNsight

    [I'll write new text for this]

    1d. Formatting a GraphML file for use with GRNsight

    [leave commented out for the moment as we await implementation]

    1e. Formatting a GML file for use with GRNsight

    [leave commented out]

2. Loading the input files

    2a. Loading a GRNsight-formatted Excel Workbook

    2b. Importing a Cytoscape SIF file

    [I'll write new text for this]

    2c. Importing a GraphML file

    [leave commented out]

    2d. Importing a GML file

    [leave commented out]

3. How GRNsight displays the graph

    3a. Unweighted and weighted graph display

    3b. Force graph layout parameters

    3c. Manually manipulating the graph

4. Interpreting the results

5. Exporting to other file formats

    5a. Exporting data to Cytoscape SIF format

    5b. Exporting data to GraphML format

    [comment out]

   5c. Exporting data to GML format

    [comment out]

6. Printing the graph

7. Further help

    [put the information under the current Further Information section here]

**Comment by kdahlquist** [comment]
I updated the outline in the previous comment to include graph statistics.

**Comment by dondi** [comment]
The outline has been implemented in `gh-pages` and as such is now live. To help me get a good sense of how the page looks in the long term, sections requested to be commented out are not yet commented out for now. I’ll comment them out before the weekend wraps up.

**Comment by kdahlquist** [comment]
I can take care of the commenting out if you want...

**Comment by dondi** [comment]
Sure, go ahead, plus edit as needed...that was a lot of cutting and pasting so I might have messed up in spots. Thanks! :)

**Comment by kdahlquist** [comment]
Made some progress in adding documentation for new features and commenting out some of the stuff that we haven't implemented yet.  Haven't tackled the hard part of describing the file formats yet.  More to come.

**Comment by kdahlquist** [comment]
All of the documentation with regards to SIF has been added to the Documentation pages, including images.  @dondi, please take a look and make sure that I was consistent with what we are implementing.

The images are at full size right now, but they can probably be made a little bit smaller to look better, but for the moment, everything is there.

Issue #316 is my request for help regarding internal links to subsections.

**Comment by dondi** [comment]
I reviewed the documentation and the content mostly corresponds to the behavior that has been uploaded to the beta page as of August 4 evening, with one exception due to a UI choice made in order to fulfill one of the specs in #309. After weighing a number of choices for how the user would choose between exporting an unweighted vs. weighted SIF when the current network is weighted, I ended up with a four-way menu layout:

* Export Data
    - To Unweighted SIF
    - To Weighted SIF
    - —————
    - To Unweighted GraphML
    - To Weighted GraphML

This choice was reached due to the following factors:
* A three-level submenu (e.g., Export Data > To SIF > Unweighted/Weighted) does not appear to be supported by Bootstrap. At least, I could not get it to do so.
* A modal/popup asking the user to choose between Unweighted vs. Weighted (when the current graph being exported is weighted) would involve more code and also more clicks in some cases.

In the end, the 4-way menu layout was the most straightforward, with no additional clicks (though perhaps more mouse movement). When an unweighted graph is on display, the "Weighted" menu items are disabled. When a weighted graph is shown, then both choices are available.

I didn't edit the documentation yet pending your view on this UI choice, but if we did stick with this, then the sections showing menu items will need to be revised.

**Comment by kdahlquist** [comment]
I'm cool with the UI choice.  It was one of the ways I would envision it to happen anyway; I tend to like having the menu option instead of having to click through windows to get to what I want to do.  I'll take care of adding it to the documentation.

**Comment by kdahlquist** [comment]
Reminders to self about what I need to work on in documentation:
* Links to download sample SIF files
* Resize sample SIF images (make a little smaller, maybe 75%)
* Explanation of Export menu options
* Replace adjacency matrix images with ones from manuscript
* Replace demo files 3 and 4 with ones used to create images

**Comment by kdahlquist** [comment]
Note to self about GraphML section: We are not using a bunch of options (nested networks, undirected networks)...

**Comment by kdahlquist** [comment]
Basics for GraphML are at least there on the page.  It needs a lot of further detail that can be pulled from the manuscript and response to reviewers, but that can wait a little.

**Comment by kdahlquist** [comment]
I have finished fleshing out the description for GraphML, section 1d.  @dondi, will you please review for accuracy and formatting.  I want to make sure that I got the descriptions right and that I used italics vs. code formatting correctly.  Thanks.

**Comment by dondi** [comment]
I’ve reviewed sections 1d (and 2c for good measure). They’re all on point, and I applied just a few tweaks for clarification and detail. If you want to see what I changed, see the highlighted portions of these commits (changes to words and phrases within a line are in bright pink/green, whereas changed lines themselves are more pastel-ish): https://github.com/dondi/GRNsight/commit/0959bf2e701f26bb8c8d7caeb36e00bf6f95af95, https://github.com/dondi/GRNsight/commit/e9a066c148581abe9f54086ebe1d8da1056f2135, and https://github.com/dondi/GRNsight/commit/62c753d7344fc9ad5e5ded24d4fef5df6db6eff2.
